test/use-checkout-pricing.test.js

```diff
@@ -0,0 +1,257 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import useCheckoutPricing from '../lib/use-checkout-pricing';
+import { withElements } from './support/helpers';
+
+const subscriptionPricingReturn = {
+  addon: jest.fn(() => subscriptionPricingReturn),
+  catch: jest.fn(() => subscriptionPricingReturn),
+  done: jest.fn(() => subscriptionPricingReturn),
+  plan: jest.fn(() => subscriptionPricingReturn),
+  tax: jest.fn(() => subscriptionPricingReturn),
+};
+
+const PRICING_METHODS = [
+  'reset',
+  'remove',
+  'reprice',
+  'addon',
+  'address',
+  'coupon',
+  'currency',
+  'giftcard',
+  'plan',
+  'shippingAddress',
+  'tax',
+];
+
+const checkoutPricingReturn = {
+  address: jest.fn(() => checkoutPricingReturn),
+  adjustment: jest.fn(() => checkoutPricingReturn),
+  catch: jest.fn(() => checkoutPricingReturn),
+  coupon: jest.fn(() => checkoutPricingReturn),
+  currency: jest.fn(() => checkoutPricingReturn),
+  done: jest.fn(cb => {
+    cb();
+    return checkoutPricingReturn;
+  }),
+  giftcard: jest.fn(() => checkoutPricingReturn),
```

Comment:
This needs to be `giftCard`

Reply:
The method right now is `giftcard`, I know there's a PR to deprecate this in favor of `giftCard`, should we switch to use this method instead? https://github.com/recurly/recurly-js/pull/580

![image](https://user-images.githubusercontent.com/32269552/74069706-a7737200-49c4-11ea-8e78-fbc3c6210591.png)